pages/index.tsx

```diff
@@ -17,27 +18,29 @@ const Home: NextPage = () => {
       <Head>
         <title>FQM Testify: an eCQM Analysis Tool</title>
       </Head>
-      <Grid>
-        <Grid.Col span={12}>
-          <MeasureUploadHeader />
-        </Grid.Col>
-        <Grid.Col span={12}>
-          <MeasureUpload />
-        </Grid.Col>
-        <Grid.Col span={12}>
-          <DateSelectors />
+      <Grid justify="center">
+        <Grid.Col span={5}>
+          <Stack justify="space-between" spacing="xl">
+            <MeasureUploadHeader />
+            <MeasureUpload />
+            <Space />
+            <DateSelectorsHeader />
+            <DateSelectors />
+            <Link href={'/generate-test-cases'}>
+              <Group position="right">
+                <Button
+                  sx={() => ({
+                    marginTop: 10
+                  })}
+                  disabled={!(measureBundle.name && measurementPeriod.start && measurementPeriod.end)}
+                >
+                  Next
```

Comment:
Should we still be allowed to actually click on the "Next" button and navigate to a new page? If I do not upload a measure bundle, the "Next" button gets disabled but I can still click on it (and then get redirected to the page about not having a measure bundle selected). I know that you can't do anything with the other disabled buttons (in the patient creation panel) so I feel like the user shouldn't be able to go to a new page when this button is disabled either? Just wanted to check if you experience the same behavior

Reply:
Nice catch!